jsvg/src/main/java/com/github/weisj/jsvg/nodes/prototype/impl/HasGeometryContextImpl.java

```diff
@@ -92,9 +102,21 @@ private HasGeometryContextImpl(@Nullable AffineTransform transform, @NotNull Len
     }
 
     @Override
-    public @NotNull Point2D transformOrigin(@NotNull MeasureContext context) {
+    public @NotNull Point2D transformOrigin(@NotNull RenderContext context) {
+        if (transformBox == TransformBox.ViewBox) {
+            MeasureContext measureContext = context.measureContext();
+            return new Point2D.Float(
+                    transformOriginX.resolve(measureContext),
+                    transformOriginY.resolve(measureContext));
+        }
+
+        @NotNull ElementBounds bounds = new ElementBounds(node, context);
+        @NotNull Rectangle2D box = transformBox == TransformBox.FillBox ? bounds.boundingBox() : bounds.strokeBox();
```

Comment:
Let's be explicit here with a switch. Also to future proof (as the `boundingBox()` method is used internally and currently doesn't follow spec semantics) let's introduce a `fillBox()` function on `ElementBounds`. For now it could simply call through to `boundingBox()`.